bench/workgen/workgen.cxx

```diff
@@ -248,6 +281,231 @@ WorkloadRunner::start_table_idle_cycle(WT_CONNECTION *conn)
     }
     return 0;
 }
+
+/*
+ * Get the cumulative size of all the files under the given directory.
+ */
+static uint32_t
+get_dir_size_mb(std::string dir)
+{
+    uint64_t result = 0;
+    for (const auto &path : std::filesystem::recursive_directory_iterator(dir)) {
+        try {
+            result += std::filesystem::file_size(path);
+        } catch (std::filesystem::filesystem_error &e) {
+            /*
+             * A file might be dropped between listing the directory contents and getting the file
+             * sizes. Ignore such errors.
+             */
+            ASSERT(std::string(e.what()).find("No such file or directory") != std::string::npos);
+            continue;
+        }
+    }
+    return result / (WT_THOUSAND * WT_THOUSAND);
+}
+
+/*
+ * This function will create "dynamic_create_count" tables every "dynamic_create_period" seconds. It
```

Comment:
I'd rather not use variable names in the comments. We can say something like "This function creates one or more tables at regular intervals, where the interval length and number of tables are specified in the workload options."